config/i18n.js

```diff
@@ -21,6 +21,15 @@ i18next.use(Backend).init({
   backend: {
     loadPath: './locales/{{lng}}/{{ns}}.json',
   },
+  interpolation: {
+    formatSeparator: '|',
+    format: function(value, format) {
```

Comment:
As discussed maybe this would be best doing it in the view layer before it hits the `i18n`. So in nunjucks use 
```
{{ t("moves::confirmation.page_title", { name: fullname | upper }) }}
```

Reply:
Have dropped this commit altogether as after discussing I don't think we need formatting here too.